cdist/conf/type/__pyvenv/man.text

```diff
@@ -0,0 +1,80 @@
+cdist-type__pyvenv(7)
+==================
```

Comment:
Attention: the length of = must equal the title length - otherwise asciidoc fails